src/App.js

```diff
@@ -1,17 +1,115 @@
-import SearchBar from './components/SearchBar/SearchBar';
-import Users from './components/Users/Users';
-import './App.css';
+import React, { useEffect, useState } from "react";
 
+import SearchBar from "./components/SearchBar/SearchBar";
+import Users from "./components/Users/Users";
+import Loading from "./components/loading/Loading";
+import Error from "./components/error/Error";
+import "./App.css";
+
+const API = "https://users-app-backend.onrender.com/users";
 function App() {
-  // TODO: Fetch data here
-
-  return (
-    <div className="App">
-      <h1>Our Users</h1>
-      <SearchBar />
-      <Users />
-    </div>
-  );
+	// TODO: Fetch data here
```

Comment:
This is a very minor point but it's also something that some engineers can be really picky about. It looks like your tab spacing is more than 2 or 4 spaces long, possibly 8? I would recommend checking your settings in VS Code and/or Prettier for this. You'll want your tabs to be 2 or 4 spaces only and make sure Prettier auto formats it that way. I _think_ I have nothing but default Prettier on my machine and it has 2 spaces for tabs.